Formula/cadaver.rb

```diff
@@ -30,46 +27,16 @@ class Cadaver < Formula
   depends_on "pkg-config" => :build
   depends_on "gettext"
   depends_on "neon"
```

Comment:
Might still have linkage to OpenSSL through neon.

Reply:
It does. I've updated the build.